lib/iris/tests/unit/fileformats/pp/test_PPDataProxy.py

```diff
@@ -47,6 +49,27 @@ def test_lnpack_raw(self):
         self.assertEqual(proxy.lbpack.n3, lbpack // 100 % 10)
         self.assertEqual(proxy.lbpack.n4, lbpack // 1000 % 10)
 
+    def test__getitem__emptyslice(self):
```

Comment:
We should have two tests, which share as much as possible, but with opposite outcomes:
- One like this, showing no file access and an empty array
- Another showing file access and a non-empty array

Reply:
:+1: